foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/generator/TargetedPipelineStepPythonGenerator.java

```diff
@@ -38,10 +38,10 @@ public void generate(GenerationContext generationContext) {
         for (Step step : pythonTargetPipeline.getSteps()) {
             if (shouldGenerateStep(step, generationContext)) {
                 VelocityContext vc = getNewVelocityContext(generationContext);
-                String profileName = MavenUtil.getPySparkDataDeliveryProfileName(generationContext);
+                boolean isRecordGenerationInPipelineModule = MavenUtil.isRecordGenerationInPipelineModule(generationContext);
                 PythonStep pythonStep = new PythonStep(step);
                 pythonStep.validate();
-                pythonStep.setProfileName(profileName);
+                pythonStep.setRecordGenerationInPipelineModule(isRecordGenerationInPipelineModule);
```

Comment:
Could simplify this to one line: `pythonStep.setRecordGenerationInPipelineModule(MavenUtil.isRecordGenerationInPipelineModule(generationContext));`

Reply:
sounds good, updated!